src/utils.ts

```diff
@@ -165,7 +165,7 @@ export interface BeginEndDatesConsistentOptions extends ValidationOptions {
 export const throwIfErrors = (errorList: string[]) => {
   if (errorList.length > 0) {
     throw new EaseyException(
-      new Error(JSON.stringify(errorList)),
+      new Error(errorList.join(' ')),
```

Comment:
This will disrupt how the errors are handled by the [`formatErrorResponse`](https://github.com/US-EPA-CAMD/easey-ecmps-ui/blob/57d471e801549950069e72c10c27af5401d6e184/src/utils/functions.js#L484) helper in the UI. Since the **`Modal`** component expects an array of error messages, I think the locations in the code should be updated to use that function, for example:
```diff
-       const errorResp = Array.isArray(resp) ? resp : [resp];
+       const errorResp = formatErrorResponse(resp);
        setErrorMsgs(errorResp);
```

```suggestion
      new Error(JSON.stringify(errorList)),
```